fhevm/fhelib.go

```diff
@@ -248,6 +248,169 @@ var fhelibMethods = []*FheLibMethod{
 		requiredGasFunction: getCiphertextRequiredGas,
 		runFunction:         getCiphertextRun,
 	},
+	// TEE operations
```

Comment:
In #31, there has been a refactor to move tee operations into teelib.go. Your new function should go there.